src/main/java/net/mcreator/ui/modgui/MusicDiscGUI.java

```diff
@@ -121,7 +121,7 @@ public MusicDiscGUI(MCreator mcreator, ModElement modElement, boolean editingMod
 		JPanel subpane2 = new JPanel(new GridLayout(6, 2, 45, 8));
 		subpane2.setOpaque(false);
 
-		name.setOpaque(false);
+		name.setOpaque(true);

```

Comment:
This is the only occurrence of modifying `opaque` parameter of VTextFields, so I think you can safely remove this line.

Reply:
Thx for the tip.